microhapulator/data/template.html

```diff
@@ -264,6 +264,13 @@ <h3>Interpretation Hints</h3>
                 </table>
             </div>
             <br />
+
+            <p> The following donut plots show the proportions of on target, repetitive, off target, and cotaminant reads. On target reads are those reads that align preferentially to the marker locus when aligned to the full human reference genome. Repetitive reads align to a marker sequence but preferentially align to a different locus when aligned to the full reference. Off target reads do not align to any marker sequences but do align elsewhere in the human genome. Lastly, contaminant reads do not align anywhere in the human genome.</p>
+       
+            {% for plot in plots["donut"] %}
+            <img src="{{plot}}" />
+            {% endfor %}
+            <br />
```

Comment:
A few recommendations here.

1. I'd suggest moving these plots directly above the "Interpretation Hints" block. The first table and these plots give a per-sample summary, while the next table gives per-marker detail for repetitive reads.
2. Following up on the last suggestion, we may consider just displaying the counts for the 4 mapped read categories in the table (on-target, repetitive, off-target, and contamination) rather than total and repetitive. With concise column labels, this shouldn't be too cluttered. There are pros and cons: showing just total and repetitive makes it very easy to spot problematic markers with repetitive sequence; but including all of the other data gives nice granularity to the data shown in the new donut plots. I'm interested in your thoughts there.
3. The plots aren't very legible when they're displayed two to a row. They'll occupy more vertical space if we have a single plot fill out the entire row, but legibility might require it.

Reply:
@standage  to clarify on point number 2, do you mean adding those columns to the table with the headings 
`Total Reads Mapped  Mapping Rate  Mapped (Chrom)  Mapping Rate (Chrom)  Chi-square`

The table with the total and repetitive counts is on a per marker level as you mentioned and the 4 mapped categories are per-sample.

Reply:
Actually, I got mixed up with point 2 above: off-target and contamination isn't relevant at the per-marker scope, only the per-sample scope. So changing "repetitive off target" to just "off target" for that table should suffice.